shell/platform/tizen/public/flutter_tizen.h

```diff
@@ -154,13 +160,27 @@ FLUTTER_EXPORT FlutterDesktopViewRef FlutterDesktopViewCreateFromNewWindow(
 
 // Creates a view that hosts and displays the given engine instance.
 //
-// The type of parent should be Evas_Object*, Cast Evas_Object* to void*.
+// The type of |parent| must be Evas_Object*.
 // @warning This API is a work-in-progress and may change.
 FLUTTER_EXPORT FlutterDesktopViewRef FlutterDesktopViewCreateFromElmParent(
     const FlutterDesktopViewProperties& view_properties,
     FlutterDesktopEngineRef engine,
     void* parent);
 
+#ifdef NUI_SUPPORT
```

Comment:
Using `#ifdef` in `flutter_tizen.h` is dangerous because this file is used for native app and plugin builds as well. But if you're really sure that the new function will only be used in C# code (`Interop.cs`), it should be okay.

Reply:
removed